build.gradle

```diff
@@ -13,7 +13,7 @@ java {
 
 allprojects {
     group = "edu.wpi.first"
-    version = "2024.7.0"
+    version = "2024.7.1"
```

Comment:
Probably more appropriate to be 2024.8.0?  @ThadHouse?

Reply:
The last time we bumped the deployment target, we did a native-utils minor version bump. https://github.com/wpilibsuite/native-utils/pull/174